lib/rucio/core/oidc.py

```diff
@@ -1444,5 +1444,5 @@ def oidc_identity_string(sub: str, iss: str):
     return 'SUB=' + str(sub) + ', ISS=' + str(iss)
 
 
-def token_dictionary(token: models.Token):
+def token_dictionary(token: models.Token) -> TokenDict:
```

Comment:
Is there something preventing us from reusing the function in `authentication.py`?